packages/modules/web-viewer-react/src/services/Initializer.ts

```diff
@@ -15,12 +15,12 @@ import {
 
 import type { IModelBackendOptions, WebViewerProps } from "../types";
 
-const getHostedConnectionInfo = async (
```

Comment:
lol. wonder why this was async? had to be a good reason at some point

Reply:
in 2.x we probably got the orch url using buddi and needed it to be async